lldb/tools/lldb-dap/Handler/ConfigurationDoneRequestHandler.cpp

```diff
@@ -52,8 +64,14 @@ void ConfigurationDoneRequestHandler::operator()(
   dap.configuration_done_sent = true;
   if (dap.stop_at_entry)
     SendThreadStoppedEvent(dap);
-  else
+  else {
+    // Client requests the baseline of currently existing threads after
+    // a successful launch or attach.
+    // A 'threads' request will be sent after configurationDone response
+    // Obtain the list of threads before we resume the process
+    dap.initial_thread_list = CacheInitialThreads(dap);
```

Comment:
Call function in JSONUtils.cpp as mentioned above, so we ensure we are using the exact same code in the 'threads' handler function below as well:
```
dap.initial_thread_list = GetThreads(dap.target.GetProcess());
```
We should also set the `dap.focus_tid` to the first thread right? We need the 'continued' event to be sent and must reliably set the `dap.focus_tid`.